scene/gui/graph_edit.cpp

```diff
@@ -587,6 +587,15 @@ void GraphEdit::_graph_element_deselected(Node *p_node) {
 	emit_signal(SNAME("node_deselected"), graph_element);
 }
 
+void GraphEdit::_graph_element_visibility_changed(Node *p_node) {
+	GraphElement *graph_element = Object::cast_to<GraphElement>(p_node);
+	ERR_FAIL_NULL(graph_element);
```

Comment:
Can be simplified to
```suggestion
void GraphEdit::_graph_element_visibility_changed(GraphElement *p_graph_element ) {
```

Reply:
You also need to change the declaration and fix formatting (the empty line needs to be removed).